### DISCUSSION TITLE
Identify if the device is connected as an advertiser

### DISCUSSION THREAD

**Discussion by MeloiMM** [shinyorg/shiny#684] [Q&A] [answered]
Is it possible to identify if a device is connected if it is the one who advertises?
The case is, I have an Android device that performs scanning and another Android that advertises. I just want to know if the latter, as an advertiser, could identify that it is connected?
Thank you!

**Comment by aritchie** [comment, marked as answer]
The API on the OS doesn't respond to "connected" until a characteristic is "worked with" whether it is subscribing, reading, or writing

**Reply by MeloiMM** [reply]
Yes thanks. You were right!
